genmd.py

```diff
@@ -290,6 +291,7 @@ def parser(txt):
                 raise Exception("Unknown MFSA header: %s" % line)
 
         # body = minidom.parseString("<html>" + bod + "</html>")
+        # print MfsaMd.xmlheader + bod + MfsaMd.xmlfooter
```

Comment:
I kept the line around because I regularly need it for debugging and I'm too lazy to write proper debug output filtering.